tests/suite/text/deco.typ

```diff
@@ -91,3 +91,6 @@ We can also specify a customized value
 #overline[Overlined]
 
 #(strike, highlight, underline, overline).fold([Mixed], (it, f) => f(it))
+
+#smallcaps[Small Caps]
```

Comment:
could you extract this into a separate test in `smallcaps.typ`? in the compiler terminology, it's not considered a "decoration".

or easiest would probably be to just annotate the `smallcaps-all` test with `render html`, i.e. `--- smallcaps-all render html ---` and make it test both.